src/main/features/dashboard/views/macro/claimStatus/partAdmissionStatesPaidAccepted.njk

```diff
@@ -0,0 +1,24 @@
+{% from "internalLink.njk" import internalLink %}
+{% from "./utilityMacros.njk" import downloadResponseLink %}
+
+{% macro partAdmissionStatesPaidClaimSettledClaimantDetails(claim) %}
+
+  <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
+  <p>{{ t('You’ve accepted the defendant’s part admission.') }}</p>
+  <p>{{ t('You agreed they owed {{ amount }} and that they’ve paid it already.',
+      {amount: claim.response.amount | numeral}) }}</p>
+
+{% endmacro %}
+
+{% macro partAdmissionStatesPaidClaimSettledDefendantsDetails(claim) %}
+
+<h2 class="heading-small">{{ t('This claim is settled') }}</h2>
```

Comment:
this is the same content as the paid in full details and could just use that?

Reply:
Paid in full macro `paidInFullDefendantClaimDetails`  is using `totalAmountTillToday` instead of `claim.response.amount`, not sure whether that is correct behaviour in that macro. @robertparkinson please verify.

Reply:
That is a good spot. I believe that paid in full may be incorrect as for a part admit we should be using the updated response amount. I will get the testers to verify

Reply:
Thank you.